libs/core/src/components/sage-tooltip/test/sage-tooltip.spec.tsx

```diff
@@ -0,0 +1,164 @@
+import { newSpecPage } from '@stencil/core/testing';
+import { SageTooltip } from '../sage-tooltip';
+
+describe('sage-tooltip', () => {
+  it('renders the trigger', async () => {
+    const { root } = await newSpecPage({
+      components: [SageTooltip],
+      html: `
+       <sage-tooltip>
+        <sage-button variant="secondary">Secondary</sage-button>
+       </sage-tooltip>`
+    });
+    expect(root).toEqualHtml(`
+      <sage-tooltip>
+        <mock:shadow-root>
+        <div class="sage-tooltip">
+          <span class="sage-tooltip__trigger">
+            <slot></slot>
+          </span>
+          <div class="sage-tooltip__content" aria-hidden="true" aria-live="off" role="tooltip">
+            <slot name="content"></slot>
+          </div>
+        </div>
+        </mock:shadow-root>
+        <sage-button variant="secondary">Secondary</sage-button>
+      </sage-link>
+    `);
+  });
+
+  it('should be able to call method to show tooltip', async () => {
+    const page = await newSpecPage({
+      components: [SageTooltip],
+      html: `
+      <sage-tooltip content="Tooltip content">
+        <sage-button variant="secondary">Secondary</sage-button>
+      </sage-tooltip>`
+    });
+    await page.root?.showTooltip();
+    await page.waitForChanges();
+
+    const element = page.root?.shadowRoot;
+
+    expect(element?.querySelector('.sage-tooltip')).toHaveClass('sage-tooltip--is-open');
+  });
+
+  it('should be able to call method to hide tooltip', async () => {
+    const page = await newSpecPage({
+      components: [SageTooltip],
+      html: `
+      <sage-tooltip content="Tooltip content">
+        <sage-button variant="secondary">Secondary</sage-button>
+      </sage-tooltip>`
+    });
+    await page.root?.showTooltip();
+    await page.waitForChanges();
+    await page.root?.hideTooltip();
+    await page.waitForChanges();
+
+    const element = page.root?.shadowRoot;
+
+    expect(element?.querySelector('.sage-tooltip')).not.toHaveClass('sage-tooltip--is-open');
+  });
+
+  it('should show arrow when hasArrow is true', async () => {
+    const page = await newSpecPage({
+      components: [SageTooltip],
+      html: `
+      <sage-tooltip hasArrow="true" content="Tooltip content">
+        <sage-button variant="secondary">Secondary</sage-button>
+      </sage-tooltip>`
+    });
+
+    const element = page.root?.shadowRoot;
+
+    expect(element?.querySelector('.sage-tooltip__arrow')).not.toBeNull();
```

Comment:
Spec's failing on this line. Does it need a focus event like the other tests below?